modules/microshift-embed-microshift-build-image.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-offline-deploy.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-embed-microshift-build-image-offline-deployment_{context}"]
+= Building the rpm-ostree image for offline deployments
+
+You can use Image Builder to create `rpm-ostree` system images with embedded {microshift-short} container images. To embed container images, you must add the image references to your Image Builder blueprint.
+
+.Prerequisites
+
+* You have root-user access to your build host.
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have created and updated an {op-system-ostree} image blueprint for offline use.
+* You have updated the `/etc/osbuild-worker/osbuild-worker.toml` configuration file for offline use.
+
+.Procedure
+
+. Import the modified blueprint TOML file into Image Builder by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli blueprints push <my_blueprint_name>
+----
+Replace `<my_blueprint_name>` with the name of your TOML blueprint file.
+
+. List existing blueprints to verify that the created blueprint is successfully pushed and exists by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli blueprints show BLUEPRINT-NAME
+----
+
+. Build the `rpm-ostree` commit with the embedded container image by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli compose start <my_blueprint_name> edge-installer
+----
+Replace `<my_blueprint_name>` with the name of your TOML blueprint file.
+
```

Comment:
Can you add this command so the user can poll when the build finishes ?

`sudo composer-cli compose status`

Reply:
I think we should just reference to another existing procedure rather than trying to replicate incomplete instructions here